PomoTodo/PomoTodo/Sources/Views/Setting/SettingViewModel.swift

```diff
@@ -9,33 +9,59 @@ import Foundation
 import SwiftUI
 import Combine
 
-class SettingViewModel: ObservableObject {
-  @Published var options: [Tag] = [
-    Tag(index: 0, name: "공부", colorId: 0),
-    Tag(index: 1, name: "취미", colorId: 1),
-    Tag(index: 2, name: "독서", colorId: 2),
-    Tag(index: 3, name: "운동", colorId: 3)
-  ]
-  @Published var timers: [PomoTimer] = [
-    PomoTimer(index: 0, focusTimeUnit: 25*60, tomatoPerCycle: 6, shortBreakUnit: 1*60, longBreakUnit: 30*60),
-    PomoTimer(index: 1, focusTimeUnit: 5*60, tomatoPerCycle: 8, shortBreakUnit: 1*60, longBreakUnit: 15*60),
-    PomoTimer(index: 2, focusTimeUnit: 60*60, tomatoPerCycle: 4, shortBreakUnit: 1*60, longBreakUnit: 60*60)
-  ]
-  @Published var pomoName: [String] = ["기본 뽀모도로","짧은 뽀모도로","긴 뽀모도로"]
+final class SettingViewModel: ObservableObject {
+  @Published var tags: [Tag] = []
+  @Published var timers: [PomoTimer] = []
+  @Published var pomoName: [String] = ["1번 프리셋","2번 프리셋","3번 프리셋"]
   
-  func optionsSave() {
-    print("options의 값을 디비에 저장")
-    // 태그 편집버튼 완료 클릭 시 실행
+  struct State {
+    var config = AppConfig(pomoTimers: [], tags: [])
```

Comment:
State 구조체는 UI에 직접 관여하는 변수를 모아두었기 때문에 `@Publish`를 쓰시면 필요없을 것 같습니다.
또한 클래스 내부에서만 쓰이는 변수이므로 private 선언해주시면 됩니다.
```suggestion
private var config = AppConfig(pomoTimers: [], tags: [])
```

Reply:
옮겨올때 무의식적으로 일단 옮겨오고 수정하자 하니 이런부분이 생기네요
다음엔 좀 더 이해를 하고 옮겨오겠습니다ㅎㅎ
수정했습니다!